cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -256,17 +256,7 @@ def verify():
     wait_for(verify, message="email receive check", delay=5)
 
 
-@pytest.mark.meta(
-    blockers=[
-        BZ(
-            1662638,
-            forced_streams=["5.9", "5.10", "upstream"],
-            unblock=lambda provider: provider.one_of(VMwareProvider)
-            and provider.version == "6.5",
-        )
-    ]
-)
-@pytest.mark.parametrize('auto', [True, False], ids=["Auto", "Manual"])
+@pytest.mark.parametrize('auto', [False], ids=["Manual"])
```

Comment:
Why did you remove "Auto" and "True" parametrization?

Reply:
Oh shoot! Committed that by mistake. Had changed it for local testing.